lib/graphql/query/serial_execution/field_resolution.rb

```diff
@@ -16,14 +16,14 @@ def initialize(ast_node, parent_type, target, query, execution_strategy)
 
         def result
           result_name = ast_node.alias || ast_node.name
-          result_value = begin
-            get_finished_value(get_raw_value)
+          raw_value = begin
+            get_raw_value
           rescue GraphQL::ExecutionError => err
             err.ast_node = ast_node
             query.context.errors << err
-            nil
+            err
           end
-          { result_name => result_value  }
+          { result_name => get_finished_value(raw_value) }
```

Comment:
Do we want `get_finished_value` to be within the `begin ... rescue` block? 

It was added intentionally here: https://github.com/rmosolgo/graphql-ruby/pull/93/files#diff-a66f6e348162887250fe0e5939b3a11bR20